### ISSUE TITLE
weird warnings

### ISSUE THREAD

**Issue by Jimmi08** [arunshekher/nofollow#2]
php 5.6

Hi, I suppose this is something with my data, but I don't know where to look.  Forum post page.

`strpos(): Empty needle, Line 428 of /***/e107_plugins/nofollow/NoFollow.php`

maybe 30 times

**Comment by arunshekher** [comment]
@Jimmi08, Yes those warnings are normal, considering the regex pattern I used. Perhaps I'll see if I can add a `if( ! empty($needle) ) {...}` conditional or similar to remove them in next release.
